srcmedia/scss/cards/_result.scss

```diff
@@ -21,19 +30,22 @@ ol.card-results {
 
     border-top: 0;
     width: $card-width;
-    /* space between cards: subtract three cards and divide remainder by 2 */;
-    margin-right: calc((100% - (#{$card-width} * 3)) / 2);
+
+    // on mobile, centered with one card per line
+    margin-right: auto;
+    margin-left: auto;
     margin-bottom: 2.8rem;
 
-    &:nth-child(3n+0) {
-        margin-right: 0;
-    }
+    // on tablet and desktop, three cards per line
+    @media (min-width: $breakpoint-s) {
+        & {
```

Comment:
is the `&` combinator supposed to have an operand here? if not, it's not doing anything.